### DISCUSSION TITLE
Increment starting from given number with postgres

### DISCUSSION THREAD

**Discussion by gmcamposano** [prisma/prisma#16430] [Q&A] [answered]
We have around 15,000 invoices so seeding all of them to be able to start at 15,001 seems kind of messy. Is there a cleaner way?

**Comment by pantharshit00** [comment, marked as answer]
Hello @gmcamposano 

Are you referring to auto incrementing column here or incrementing any other integer field?

**Reply by gmcamposano** [reply]
I am referring to autoincrement column starting from a given number.

**Reply by pantharshit00** [reply]
In that case, you can run the following query:
```ts
await prisma.$executeRaw`ALTER SEQUENCE "User_id_seq" RESTART 15000;`
```

Replace `User_id_seq` to the autoincrementing sequence you want to alter. If you are using migrate, it following `<model>_<field>_seq` convention but you can also view this with a database GUI like tableplus.io or https://dbeaver.io/

Please do mind, you might get a unique constraints error on ID field if you reset the sequence to certain value and the next autoincrementing value from the altered sequence matches any existing row ID in your table.

**Reply by gmcamposano** [reply]
Thank you @pantharshit00

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
